src/adapters/base.ts

```diff
@@ -40,6 +44,12 @@ interface IRepoAdapter {
   getBaseBranch(repo: IRepo): string;
 
   getEnvironmentVariables(repo: IRepo): Promise<IEnvironmentVariables>;
+
+  createIssue(repo: IRepo): Promise<string>;
```

Comment:
Consider exposing issue interface with repo as required and issueNumber as optional.  If issueNumber exists, update.  Otherwise, create a new issue.